src/QueryHandler/QueryHandler.php

```diff
@@ -102,7 +102,7 @@ public function queryAndReturnResult(GraphQLSchema $schema, $query, ?array $vars
             return GraphQL::executeQuery($schema, $query, null, $context, $vars);
         };
 
-        return $this->callMiddleware($schema, $query, $context, $vars, $last);
+        return $this->callMiddleware($schema, $query, $context, (array) $vars, $last);
```

Comment:
```suggestion
        return $this->callMiddleware($schema, $query, $context, $vars ?? [], $last);
```
This is the syntax we've used elsewhere in core code for this purpose - we should continue following this pattern.